Uplift/Views/Headers/HomeSectionHeaderView.swift

```diff
@@ -93,6 +93,6 @@ class HomeSectionHeaderView: UICollectionReusableView {
             "Section": "\(HomeViewController.SectionType.todaysClasses.rawValue)/viewAll"
             ])
 
-        buttonCompletion?()
+        buttonCompletion!()
```

Comment:
Why do we make the closure optional here? Is it safe to force-unwrap this? Can we make it non-optional instead?

Reply:
I left the ! in there because the buttonCompletion() function didn't seem like it was executing, but I'll change it back to optional for now